From f316f413f589989013bf161170087e77a82db2d3 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Sat, 5 Nov 2016 16:38:14 +0100
Subject: [PATCH] remove OpenGL transitions config. file from scheme

The OpenGL transitions are optional, so the respective config. file
should only be listed in the ogltrans module.

This reverts commit dafcb2844bda4af2bc290548548392bda3262307.

Change-Id: Ie785db07d04a0baf3dc9fd385f1d8a34837dc38b
---
 officecfg/registry/schema/org/openoffice/Office/Impress.xcs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/officecfg/registry/schema/org/openoffice/Office/Impress.xcs b/officecfg/registry/schema/org/openoffice/Office/Impress.xcs
index e8b170d..323e7ef 100644
--- a/officecfg/registry/schema/org/openoffice/Office/Impress.xcs
+++ b/officecfg/registry/schema/org/openoffice/Office/Impress.xcs
@@ -452,7 +452,7 @@
           </desc>
           <label>List of files containing transition effects</label>
         </info>
-        <value oor:separator=";">vnd.sun.star.expand:$BRAND_BASE_DIR/$BRAND_SHARE_SUBDIR/config/soffice.cfg/simpress/transitions.xml;vnd.sun.star.expand:$BRAND_BASE_DIR/$BRAND_SHARE_SUBDIR/config/soffice.cfg/simpress/transitions-ogl.xml</value>
+        <value oor:separator=";">vnd.sun.star.expand:$BRAND_BASE_DIR/$BRAND_SHARE_SUBDIR/config/soffice.cfg/simpress/transitions.xml</value>
       </prop>
       <prop oor:name="EffectFiles" oor:type="oor:string-list" oor:nillable="false">
         <!-- OldPath: -->
-- 
2.9.3

